client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
further: would this work as `func (snap *Snap) Price(currency string) string`? that is, as a method that returned the price, instead of setting it?